user-manual/transfer/dataverse.rst

```diff
@@ -0,0 +1,118 @@
+.. _dataverse-transfers:
+
+=====================
+Dataverse Integration
+=====================
+
+.. _dataverse-overview:
+
+Overview
+--------
+
+`Dataverse`_ is an open source repository for research data. Archivematica can
+be configured to use a Dataverse Repository as a Transfer Source Location. 
+Dataverse Transfer Source Locations can be configured to display all available 
+datasets or a subset using query parameters.
```

Comment:
Suggested: `subset of them`.

**NB.** Depending on how a query is configured a user will only see the first 50 results returned.